pkgs/by-name/na/nautilus/package.nix

```diff
@@ -100,6 +101,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   mesonFlags = [
     "-Ddocs=true"
+    "--cross-file=${writeText "crossfile.conf" ''
```

Comment:
It only appears to be used for tests:

https://gitlab.gnome.org/GNOME/nautilus/-/blob/9d5769c73d82280129ab3ff1156d3d9f2eac625d/test/automated/displayless/meson.build#L1

Perhaps it would be better to disable tests since we do not run them anyway.

https://gitlab.gnome.org/GNOME/nautilus/-/blob/9d5769c73d82280129ab3ff1156d3d9f2eac625d/meson_options.txt#L58

Reply:
`localsearch` ends up in the non-test output, even for native builds. `$out/bin/nautilus` is wrapped with `XDG_DATA_DIRS=${localsearch}/share/gsettings-schemas/localsearch-3.8.2:...`. same with `$out/bin/nautilus-autorun-software`.

so disabling this part of the patch means Tracker-related schemas (localsearch) won't be available to the cross-compiled nautilus. i have no idea if that matters or not: do you think it's worth it?

Reply:
But wrapping is done by our setup hook ­– I do not see how crossfile is relevant.

Reply:
oh, you're right. pushed a new patch: no cross-file mention anymore, just `-Dtests=...`.